src/js/components/Grommet/doc.js

```diff
@@ -25,6 +25,14 @@ export const doc = Grommet => {
         control scrolling.`,
       )
       .defaultValue(false),
+    options: PropTypes.object
+      .description(
+        `Additional configurations that will be applied throughout the 
+        application. If options.layer.singleId === true, id on Layer will be 
+        placed on StyledLayer and '{id}__container' will be placed on 
```

Comment:
I think the terminology of StyledLayer is confusing to the reader since users that are not familiar with the source code wouldn't understand what it is referring to.

Reply:
Good feedback. Let me work on the wording